src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  @extend %calendar_size;
```

Comment:
No need for creating %calendar_size if it is used only once. Put css properties directly here